sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -908,15 +1034,27 @@ export type IndexerExecutionStatus = "transientFailure" | "success" | "inProgres
 // @public
 export type IndexerExecutionStatusDetail = string;
 
+// @public
+export type IndexerPermissionOption = string;
+
+// @public
+export type IndexerResyncOption = string;
+
+// @public
+export interface IndexersResyncOptionalParams extends coreClient.OperationOptions {
+}
+
 // @public
 export interface IndexerState {
```

Comment:
Were the renamed properties GA'd or from a previous beta?

Reply:
`IndexerState` is in preview, and the renamed property in `IndexerExecutionResult` is in preview as well.